Integrations/CheckPhish/CheckPhish.py

```diff
@@ -0,0 +1,212 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+BASE_URL = demisto.params()['url']
+API_KEY = demisto.params().get('token')
+GOOD_DISP = demisto.params().get('good_disp')
+SUSP_DISP = demisto.params().get('susp_disp')
+BAD_DISP = demisto.params().get('bad_disp')
+USE_SSL = not demisto.params().get('insecure', False)
+handle_proxy('proxy')
```

Comment:
no need to send the proxy param, and please move this section to the ''' COMMANDS MANAGER / SWITCH PANEL ''' section